src/UI/class-settings-page.php

```diff
@@ -1094,7 +1186,7 @@ private static function get_logo_default(): string {
 	 * @param array $array Array of options to be sanitized.
 	 * @return array
 	 */
-	private static function validate_option_array( array $array ): array {
+	private static function sanitize_option_array( array $array ): array {
```

Comment:
Why the name change?

Reply:
This was done for clarity, as this function doesn't validate, but rather _**sanitizes**_ the array passed to it. As such, `sanitize` is much more descriptive of what the function does in reality, and makes it stand out from the functions starting with `validate`.

As this is a private function, changing it wasn't risky, so I went ahead with it.